conn_others_test.go

```diff
@@ -25,6 +25,11 @@ func TestOthersConnUnimplemented(t *testing.T) {
 			want, got)
 	}
 
+	if got := c.SendMessages([]Message{Message{}}); want != got {
```

Comment:
Nit: just pass `nil` for simplicity since it'll always return an error anyway.

Reply:
yes, agree